t/utils/utils.go

```diff
@@ -336,10 +337,43 @@ func ImportTestAccount(keystoreDir, accountFile string) error {
 	}
 
 	dst := filepath.Join(keystoreDir, accountFile)
-	err := ioutil.WriteFile(dst, static.MustAsset("keys/"+accountFile), 0644)
+	err := copyFile(dst, path.Join(params.GetStatusHome(), "static/keys/", accountFile))
 	if err != nil {
 		logger.Warn("cannot copy test account PK", "error", err)
 	}
 
 	return err
 }
+
+func getTestConfigFromFile(fileName string, config *testConfig) error {
+	configData, err := ioutil.ReadFile(fileName)
+	if err != nil {
+		return err
+	}
+
+	if err := json.Unmarshal(configData, &config); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func copyFile(src, dst string) error {
+	in, err := os.Open(src)
+	if err != nil {
+		return err
+	}
+	defer in.Close()
+
+	out, err := os.Create(dst)
+	if err != nil {
+		return err
+	}
+	defer out.Close()
+
+	_, err = io.Copy(out, in)
+	if err != nil {
+		return err
+	}
+	return out.Close()
```

Comment:
Shouldn't it be `return nil`? We call `defer out.Close()` a few lines earlier.

Reply:
And that's done too